Makefile

```diff
@@ -100,16 +100,22 @@ prod-build:
 
 prod-build-android: export TARGET_OS ?= android
 prod-build-android:
-	lein prod-build-android
+	cp params_prod.edn params.edn && \
+	lein prod-build-android && \
+	node prepare-modules.js
 
 prod-build-ios: export TARGET_OS ?= ios
 prod-build-ios:
-	lein prod-build-ios
+	cp params_prod.edn params.edn && \
+	lein prod-build-ios && \
+	node prepare-modules.js
```

Comment:
Can we delete `params.edn` at the end?

Reply:
nope, it should be changed before cljs compilation

Reply:
and can't we delete that file after cljs is compiled?

Reply:
I'm slow. I'll try to move it to env vars now because it seems it's a pain in the ass at the moment.

Reply:
Thanks, it does look easier to integrate with Nix now.

Reply:
ok, great